corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/entities/menu.js

```diff
@@ -99,6 +99,13 @@ FormplayerFrontend.module("Entities", function (Entities, FormplayerFrontend, Ba
                 success: function (request) {
                     defer.resolve(request);
                 },
+                error: function (request) {
+                    FormplayerFrontend.request(
+                        'error',
+                        gettext('Unable to connect to form playing service')
```

Comment:
assume this gets shown to a user? could make it more actionable... e.g. "please report an issue if you continue to see this message"